packages/waku/src/lib/builder/serve-aws-lambda.ts

```diff
@@ -3,16 +3,25 @@ import { handle } from 'hono/aws-lambda';
 import { serveStatic } from '@hono/node-server/serve-static';
 
 import { honoMiddleware } from '../middleware/hono-prd.js';
+import path from 'node:path';
+import { existsSync } from '../utils/node-fs.js';
+import { readFileSync } from 'node:fs';
```

Comment:
It's tricky, but we can use the normal one:
```suggestion
import { existsSync, readFileSync } from 'node:fs';
```